src/ray/raylet/node_manager.cc

```diff
@@ -1750,10 +1750,41 @@ void NodeManager::SubmitTask(const Task &task, const Lineage &uncommitted_lineag
   RAY_LOG(DEBUG) << "Submitting task: " << task.DebugString();
 
   if (local_queues_.HasTask(task_id)) {
-    RAY_LOG(WARNING) << "Submitted task " << task_id
-                     << " is already queued and will not be reconstructed. This is most "
-                        "likely due to spurious reconstruction.";
-    return;
+    bool ignore_task = true;
```

Comment:
I think @ericl is right, this change is not needed after https://github.com/ray-project/ray/pull/6280.

Also, it does look unnecessary to put the task in swap queue in the first place.

Reply:
ok, updated with the suggested approach